Cython/Utility/ObjectHandling.c

```diff
@@ -1945,6 +1945,11 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_FastCall(PyObject *func, PyObject
     if (f) {
         return f(func, args, nargs, NULL);
     }
+    #elif __Pyx_CyFunction_USED && CYTHON_BACKPORT_VECTORCALL
+    if (Py_TYPE(func) == __pyx_CyFunctionType) {
```

Comment:
I assume this is spelled out because the fused functions subtype is excluded?

Reply:
Yes indeed.